src/platform/site-wide/exportsFile.js

```diff
@@ -14,7 +14,7 @@ export {
 
 export { FeatureToggleReducer } from './feature-toggles/reducers';
 
-export { default as header } from './header';
+// export { default as header } from './header';
```

Comment:
Should these commented exports be removed?

Reply:
@randimays I have that as a TO DO. These were systematically commented out to help us firgure out which imports are causing problems.

Reply:
Got it 👍